frontend/js/plugins/Coreto_QuestVN.js

```
@@ -270,7 +271,7 @@
         const owner = { module: PLUGIN_NAME, questKey, entryKey, originMapId: origin.mapId, originEventId: origin.eventId };
         const token = FlowCoordinator.acquire("questVN", owner, {
             blockMovement: true,
-            blockMenu: true,
+            blockMenu: false,
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)